build.gradle

```diff
@@ -51,8 +50,7 @@ dependencies {
     compile files('lib/threetenbp-1.3.1.jar')
     compile files('lib/LGoodDatePicker-10.3.1-backport.jar')
 
-    runtime group: 'org.slf4j', name: 'jcl-over-slf4j', version: '1.7.5'
-    runtime group: 'org.slf4j', name: 'slf4j-simple', version: '1.7.5'
```

Comment:
I don't get any error when I removed the jcl-over-slf4j or both of this dependencies. I added later because I talked in slack and was deemed needed to have explicit dependency to jcl-over-slf4j instead of relying on the one comes from google-analytics-java dependency.

Reply:
@nribeka your comment actually highlights my point. You just created changes and make it look like original work. This was already fixed in 9c8cbd0ddd43.

Reply:
By the way, I don't disagree with the explicit dependency (as you can see, I actually added it). I'm not sure what was said on slack about it, but when your code actually depends on transitives it adds risk that changes made to downstream libraries breaks the build with no apparent change (the build file won't change, just the artifact metadata, which is only visible in the artifact repository). Also, by modeling the *actual* dependencies the build file tells you a story. Lots of deps, your project will be more susceptible to decay over time and may suffer from library mismatching or conflicts.